### PR TITLE
Add sensors about cpu, memory and disk utilisation to proxmoxve

### PR DESCRIPTION
**Pull request by mib1185**
## Proposed change

Add many new sensors about cpu, memory and disk utilisation to proxmoxve integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#42811
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/proxmoxve.markdown [hunk 1]
```diff
@@ -100,9 +101,39 @@ proxmoxve:
 
 ## Binary Sensor
 
-The integration will automatically create a binary sensor for each tracked virtual machine or container. The binary sensor will either be on if the VM's state is running or off if the VM's state is different.
+The integration will automatically create a binary sensor for each tracked virtual machine or container. The binary sensor will either be on if the VM's state is running or off if the VM's state is different. There are als additional attributes added to each binary sensor for the node running on, VM name, VM id, VM type, CPU count and memory size.
```
Comment (frenck) [inline source/_integrations/proxmoxve.markdown]:
```suggestion
The integration will automatically create a binary sensor for each tracked virtual machine or container. The binary sensor will either be on if the VM's state is running or off if the VM's state is different. There are also additional attributes added to each binary sensor for the node running on, VM name, VM id, VM type, CPU count and memory size.
```

File: source/_integrations/proxmoxve.markdown [hunk 2]
```diff
@@ -100,9 +101,39 @@ proxmoxve:
...
+
+The created sensor will be called `binary_sensor.NODE_VMNAME_running`.
```
Comment (frenck) [inline source/_integrations/proxmoxve.markdown:104]: This should not be needed, as they show up as devices?

File: source/_integrations/proxmoxve.markdown [hunk 3]
```diff
@@ -100,9 +101,39 @@ proxmoxve:
...
+
+example for additional attributes
+```yaml
+node: pve
+vmid: 207
+vmname: devhost
+type: lxc
+cpu: 8
+memory: 16384
+```
```
Comment (frenck) [inline source/_integrations/proxmoxve.markdown]: Additional attributes? Why aren't they sensors?

File: source/_integrations/proxmoxve.markdown [hunk 4]
```diff
@@ -100,9 +101,39 @@ proxmoxve:
...
+
+## Sensor
+
+The integration will automatically create multiple sensors for each tracked virtual machine or container about their cpu, memory and disk utilisation.
```
Comment (frenck) [inline source/_integrations/proxmoxve.markdown]:
```suggestion
The integration will automatically create multiple sensors for each tracked virtual machine or container about their CPU, memory and disk utilization.
```

File: source/_integrations/proxmoxve.markdown [hunk 5]
```diff
@@ -100,9 +101,39 @@ proxmoxve:
...
+
+The created sensors will be called `sensor.NODE_VMNAME_SENSORNAME`.
```
Comment (frenck) [inline source/_integrations/proxmoxve.markdown:118]: This line should not be needed, as they can be found via the devices and customized in the UI.

File: source/_integrations/proxmoxve.markdown [hunk 6]
```diff
@@ -100,9 +101,39 @@ proxmoxve:
...
+| cpu_use_percent | % | CPU use | lxc, qemu |
+| cpu | None | CPU count | lxc, qemu |
+| memory_free | MiB | Memory free | lxc, qemu |
+| memory_use | MiB | Memory use | lxc, qemu |
+| memory_use_percent | % | Memory use (percent) | lxc, qemu |
+| mem | MiB | Memory size | lxc, qemu |
+| disk_free | GiB | Disk free | lxc |
+| disk_use | GiB | Disk use | lxc |
+| disk_use_percent | % | Disk use (percent) | lxc |
+| disk | GiB | Disk size | lxc, qemu |
```
Comment (frenck) [inline source/_integrations/proxmoxve.markdown]:
```suggestion
| sensor name | unit of measurement | description | available for |
| --- | --- | --- | --- |
| cpu_use_percent | % | CPU usage | LXC, QEMU |
| cpu | None | CPU count | LXC, QEMU |
| memory_free | MiB | Memory free | LXC, QEMU |
| memory_use | MiB | Memory use | LXC, QEMU |
| memory_use_percent | % | Memory usage (percent) | LXC, QEMU |
| mem | MiB | Memory size | LXC, QEMU |
| disk_free | GiB | Disk free | LXC |
| disk_use | GiB | Disk usage | LXC |
| disk_use_percent | % | Disk usage (percent) | LXC |
| disk | GiB | Disk size | LXC, QEMU |
```

File: source/_integrations/proxmoxve.markdown [hunk 7]
```diff
@@ -13,7 +14,7 @@ ha_domain: proxmoxve
 
 [Proxmox VE](https://www.proxmox.com/en/) is an open-source server virtualization environment. This integration allows you to poll various data from your instance.
 
-After configuring this component, the binary sensors automatically appear.
+After configuring this component, the sensors automatically appear.
```
Comment (frenck) [inline source/_integrations/proxmoxve.markdown]: That is assumed and not needed to be added here.

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @mib1185!

I've left some comments, could you take a look? Thanks already! 👍

File: source/_integrations/proxmoxve.markdown (hunk 2 above)
Comment (mib1185) [reply source/_integrations/proxmoxve.markdown:104]: for now these are no devices, but simple binary sensors with attributes.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
Closing this PR as the parent PR has been closed.

**Pull request closed without merging**